testkit/appkit/app.go

```diff
@@ -113,11 +114,18 @@ func GetBlockHashByHeight(ip net.IP, height int) (string, error) {
 		return "", err
 	}
 
-	var res coretypes.ResultBlock
-	if err := json.Unmarshal(body, &res); err != nil {
+	var rpcResponse types.RPCResponse
+	if err := rpcResponse.UnmarshalJSON(body); err != nil {
 		return "", err
 	}
-	return res.BlockID.Hash.String(), nil
+
+	var resBlock coretypes.ResultBlock
+	if err := tmjson.Unmarshal(rpcResponse.Result, &resBlock); err != nil {
+		return "", err
+	}
+
+	fmt.Println(resBlock.BlockID.Hash.String())
```

Comment:
do we want to keep print statements here?

Reply:
Removed